src/components/Spinner.js

```diff
@@ -0,0 +1,21 @@
+import classnames from 'classnames';
+
+/**
+ * @typedef SpinnerProps
+ * @prop {string} [classes] - Additional CSS classes to apply
+ */
+
+/**
+ * Loading indicator.
+ *
+ * @param {SpinnerProps} props
+ */
+export function Spinner({ classes = '' }) {
+  return (
+    <span className={classnames('Hyp-Spinner', classes)}>
```

Comment:
Currently LMS uses an SVG. I don't have a strong preference for either, but I think the svg was added after the css spinner in the client. We could discuss tomorrow and pick one or othe other and then replace as we move forward.